keptn/keptn
# fix(distributor): Parsing of url forces scheme to http or https

### Related Issues


Fixes #7577

### How to test

set a distributor to an image based on this PR, add the KEPTN_API_ENDPOINT to be a malformed address, check distributor output
![image](https://user-images.githubusercontent.com/89971034/166473756-208732d1-e809-43ec-8506-e803e4766031.png)

![image](https://user-images.githubusercontent.com/89971034/166631706-3ee97f79-4e94-4249-9d56-fb9139987194.png)
